templates/jetty.xml.erb

```diff
@@ -31,24 +31,6 @@ DTD Configure//EN" "http://jetty.mortbay.org/configure.dtd">
     <!-- Use this connector for many frequently idle connections and for
         threadless continuations. -->
     <Call name="addConnector">
-        <Arg>
-            <New class="org.eclipse.jetty.server.nio.SelectChannelConnector">
-                <Set name="host"><%= scope.lookupvar('opendaylight::odl_bind_ip') %>
-                    <Property name="jetty.host" />
-                </Set>
-                <Set name="port">
-                    <Property name="jetty.port" default="8181" />
-                </Set>
-                <Set name="maxIdleTime">300000</Set>
-                <Set name="Acceptors">2</Set>
-                <Set name="statsOn">false</Set>
-                <Set name="confidentialPort">8543</Set>
-                <Set name="lowResourcesConnections">20000</Set>
-                <Set name="lowResourcesMaxIdleTime">5000</Set>
-            </New>
-        </Arg>
-    </Call>
-    <Call name="addConnector">
```

Comment:
as for the purpose of this change, I'm not 100% either.

@trozet, what do we want here? you just want to stop 8181 from lighting up, or you want to also make 8081 come on in place of 8181. From what I know, there are two places that control this:

etc/jetty.xml
etc/org.ops4j.pax.web.cfg

Where is tripleO or apex, etc controlling this 8081?